src/screens/loaders/CardLoader.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ShowCardLoader, ShowListCardLoader } from '../../components';
+import useWindowSize from '../../hooks/useWindowSize';
+
+interface CardLoaderProps {
+    query?: string | null;
```

Comment:
Is `query` used anywhere here? If not, can we remove it?

Reply:
remove the unnecessary variable.